### PR TITLE
feat(store): add mediamarkt austria

### PR DESCRIPTION
**Pull request by mreilaender**
### Description

Adds support for [mediamarkt.at](https://www.mediamarkt.at/)

### Testing

- Included added store to the configuration, replaced a link from a graphics card with a product that exists and is available
- started the application and the graphic card model of the replaced link was shown in the logs

### CODE REVIEW

**Review by jef — DISMISSED** [review]
Looks good to me! Thank you for the contribution! 💯

File: src/store/model/mediamarkt-at.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      },
+    ],
+  },
+  links: [
+    {
+      brand: 'test:brand',
+      model: 'test:model',
+      series: 'test:series',
+      url: 'https://www.mediamarkt.de/de/product/-2641856.html',
+    },
```
Comment (jef) [inline src/store/model/mediamarkt-at.ts:44]: I would change this to use mediamarkt.at as well if you don't mind.

File: src/store/model/mediamarkt-at.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    {
+      brand: 'gainward',
+      model: 'phoenix',
+      series: '3060ti',
+      url:
+        'https://www.mediamarkt.at/de/product/_gainward-grafikkarte-geforce-rtx-3060-ti-phoenix-golden-sample-8gb-ga-3060rtx-ti-ph-gs-1815563.html',
```
Comment (jef) [inline src/store/model/mediamarkt-at.ts:50]:
```suggestion
        'https://www.mediamarkt.at/de/product/-1815563.html',
```

Also seems we can normalize these urls by using this. Do you mind changing that?

Comment (mreilaender) [reply src/store/model/mediamarkt-at.ts]: Done

**Review by jef — COMMENTED** [review]
Awesome! Everything looks great. Do you mind adding this into the documentation?

Thank you!

**Comment by mreilaender** [comment]
@jef The list of supported stores is all I have to add to the doc right?

Also thanks for your awesome work. Your code looks absolutely gorgeous :-)

**Review by jef — APPROVED** [review]
Great! Thank you!

**Pull request merged**